lib/utils/tts-streaming-buffer.js

```diff
@@ -377,14 +377,14 @@ class TtsStreamingBuffer extends Emitter {
 
   _onTimeout() {
     this.logger.debug('TtsStreamingBuffer:_onTimeout Timeout waiting for sentence boundary');
+    this.timer = null;
     // Check if new text has been added since the timer was set.
     const now = Date.now();
     if (now - this.lastUpdateTime < TIMEOUT_RETRY_MSECS) {
```

Comment:
Hi @davehorton 
TIMEOUT_RETRY_MSECS = 3000
There is a case that the last token came shortly and then this timeout happen.
jambonz reset timer for another 3 seconds. and no more token come.

This make user wait for 3 seconds to hear back the audio. should we reduce this timeout to 1 second?

Reply:
agree

Reply:
fixed